evogym/simulator/SimulatorCPP/Environment.h

```diff
@@ -17,6 +17,12 @@
 using namespace std;
 using namespace Eigen;
 
+#ifdef __APPLE__
+using RefMatrixXd = Matrix <double, 2, Dynamic>;
+#else
+using RefMatrixXd = Ref <MatrixXd>;
+#endif
```

Comment:
it is switching types here. Although I was not able to determine the correct way to fix it, I was able to confirm that it is possible to build in this state.

Reply:
This is a bit jank but I think this solution is better than the alternative of broken code.